### ISSUE TITLE
Queue commands list until player has fully joined.

### ISSUE THREAD

**Issue by crashdemons** [crashdemons/AztecTabCompleter#7]
Task described by title.  May relieve Issue #6

**Comment by crashdemons** [comment]
Attempted implementation in 93ab62bf26f6bfe5fe8b2674b66eeeda4087e11a

Use tests didn't seem to work, user didn't receive the commands.

So I should work on:
- [x] check that onPlayerJoin is implemented properly
- [x] verify that PlayerJoinEvent is fired reasonably after COMMANDS packet in general use
- [x] investigate possibly instead using another task that just delays sending the packet until the player is online, since changing dimension and other things can potentially trigger new COMMANDS packets.
- [x] convert filtered packet queue to unfiltered packet queue and do permissions checks in join/task
- [x] finalize implementation
- [x] complete use testing.

**Comment by crashdemons** [comment]
Blocked on Issue #8 - requires changes to queuing.
EDIT: unblocked.
